test/e2e/crcsuite/ux/uxsuite.go

```diff
@@ -121,6 +129,10 @@ func guaranteeClusterState(state string) error {
 	return notificationHandler.ClearNotifications()
 }
 
+func clusterIsOnState(expectedState string) error {
```

Comment:
using `On` is probably not the right word here. preferable use `clusterHasState` or `clusterIsInState`, as `on` can mean activated.
Since you compare the cluster state with a given value, `clusterEqualsState()` might also work

Reply:
ack

Reply:
Changed as per suggestion